src/components/config.tsx

```diff
@@ -44,6 +44,7 @@ export default (): JSX.Element | null => {
       return
     }
     // we get the iframe origin from a query parameter called 'origin', if this is loaded in an iframe
+    // TODO: why we need this origin here? where is targetOrigin used?
     const targetOrigin = decodeURIComponent(window.location.search.split('origin=')[1])
```

Comment:
@SgtPooki is this something we want to keep, or can it be removed?
iirc if we don't have query param, we would reuse cache, making subdomain page loads faster

Reply:
This is only for loading config in iframe. Iframe needs to know parent origin to postmessage

Reply:
So if we can do that another way, we could remove this

Reply:
Could we use `#hash` instead? It does not get sent to server, but JS in inframe should see it.